implementations/rust/ockam/ockam_command/src/policy/delete.rs

```diff
@@ -43,21 +44,22 @@ async fn run_impl(
     opts: CommandGlobalOpts,
     cmd: DeleteCommand,
 ) -> miette::Result<()> {
-    let node = parse_node_name(&cmd.at)?;
+    let at = get_node_name(&opts.state, &cmd.at);
+    let node_name = parse_node_name(&at)?;
     if opts
         .terminal
         .confirmed_with_flag_or_prompt(cmd.yes, "Are you sure you want to delete this policy?")?
     {
         let req = Request::delete(policy_path(&cmd.resource, &cmd.action));
-        let mut rpc = Rpc::background(ctx, &opts, &node)?;
+        let mut rpc = Rpc::background(ctx, &opts, &node_name)?;
         rpc.request(req).await?;
         rpc.is_ok()?;
 
         opts.terminal
             .stdout()
-            .plain(fmt_ok!("Policy with name '{}' has been deleted", &cmd.at))
-            .machine(&cmd.at)
-            .json(serde_json::json!({ "policy": { "at": &cmd.at } }))
+            .plain(fmt_ok!("Policy with name '{}' has been deleted", &at))
+            .machine(&at)
+            .json(serde_json::json!({ "policy": { "at": &at} }))
```

Comment:
Here you should be using the `node_name` variable, as the `at` variable could contain a `MultiAddr` formatted string (e.g. `/node/n1`, when we actually want to display just the `n1` part).

Reply:
@adrianbenavides Done! :)